### PR TITLE
UHM-7070: Lab OWA test result not showing short names

### PR DESCRIPTION
**Pull request by mogoodrich**
SL-307: Cancelling orders from display page doesn't reload labs properly

### CODE REVIEW

**Comment by mogoodrich** [comment]
This PR includes:

https://pihemr.atlassian.net/browse/UHM-7070

https://pihemr.atlassian.net/browse/SL-307

File: app/js/actions/labOrdersAction.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ export const fetchLabOrders = (testOrderType, options) => {
...
+    payload: axiosInstance.get(`order?s=default&totalCount=true&sort=desc&excludeDiscontinueOrders=true&orderTypes=${testOrderType}&activatedOnOrAfterDate=${moment(options.dateFromField).format('YYYY-MM-DD')}&activatedOnOrBeforeDate=${moment(options.dateToField).format('YYYY-MM-DD')}&excludeCanceledAndExpired=${options.excludeCanceledAndExpired}&canceledOrExpiredOnOrBeforeDate=${options.canceledOrExpiredOnOrBeforeDate ? moment(new Date()).format('YYYY-MM-DD') : ''}&fulfillerStatus=${options.fulfillerStatus ? options.fulfillerStatus : ''}&includeNullFulfillerStatus=${includeNullFulfillerStatus !== null ? includeNullFulfillerStatus : ''}&concepts=${options.conceptUuids ? options.conceptUuids : ''}&accessionNumber=${options.accessionNumber ? options.accessionNumber : ''}&patient=${options.patient ? options.patient : ''}&v=${ORDER_REP}&limit=${pageSize}&startIndex=${options.page ? options.page * pageSize : 0}`),
```
Comment (mogoodrich) [inline app/js/actions/labOrdersAction.js:30]: if options.accessionNumber was undefined here, it was passing in "accessionNumber=undefined, and the search wasn't returning anything.

File: app/js/sagas/labOrdersSaga.js [hunk 2]
```diff
@@ -187,9 +186,7 @@ function* updateOrders() {
   const labResultsTestOrderType = selectProperty(state, 'labResultsTestOrderType');
   const ordersBatchSize = selectProperty(state, 'ordersBatchSize');
   const options = {
-    dateToField: moment(labOrdersListFilters.dateToField).format('YYYY-MM-DD'),
-    dateFromField: moment(labOrdersListFilters.dateFromField).format('YYYY-MM-DD'),
-    excludeCanceledAndExpired: true,
+    ...labOrdersListFilters,
     ordersBatchSize: (ordersBatchSize || DEFAULT_ORDERS_BATCH_SIZE),
```
Comment (mogoodrich) [inline app/js/sagas/labOrdersSaga.js:190]: I'm not sure why we were not passing in all the filters here, but the net affect was that after reloading after cancelled an order, it wouldn't use the proper filters on it's search... (I'm guessing this was old code we forgot to update after the filters were added?)

File: app/js/sagas/patientSaga.js [hunk 3]
```diff
@@ -13,7 +13,7 @@ import actionTypes, { FETCH_PATIENT_LAB_TEST_RESULTS } from '../actions/actionTy
 import patientAction from '../actions/patientAction';
 import { selectProperty } from '../utils/globalProperty';
 
-const conceptRep = `(uuid,display,name,conceptClass:(uuid,display,name),datatype:(uuid,display,name),units)`;
```
Comment (mogoodrich) [inline app/js/sagas/patientSaga.js:16]: needed to return names here, not just the name, so that the getConceptShortName method would work

**Review by cioan — APPROVED** [review]
LGTM. Thanks @mogoodrich !

**Pull request merged**